<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="INSGH045PMS0" name="OracleLoad.ChkFileExitsUnixCmd" tests="1" failures="0" timestamp="2020-05-08T20:24:10 IST" time="2.861" errors="1">
  <testcase name="Filechkunix" time="2.861" classname="OracleLoad.ChkFileExitsUnixCmd">
    <error type="java.lang.RuntimeException" message="failed to create file exits.">
      <![CDATA[java.lang.RuntimeException: failed to create file exits.
at OracleLoad.ChkFileExitsUnixCmd.Filechkunix(ChkFileExitsUnixCmd.java:68)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: 4: 
at com.jcraft.jsch.ChannelSftp.ls(ChannelSftp.java:1747)
at com.jcraft.jsch.ChannelSftp.ls(ChannelSftp.java:1553)
at OracleLoad.ChkFileExitsUnixCmd.Filechkunix(ChkFileExitsUnixCmd.java:55)
... 28 more
Caused by: java.lang.NullPointerException
at com.jcraft.jsch.ChannelSftp.ls(ChannelSftp.java:1570)
... 30 more
]]>
    </error>
  </testcase> <!-- Filechkunix -->
  <system-out/>
</testsuite> <!-- OracleLoad.ChkFileExitsUnixCmd -->
